### DISCUSSION TITLE
Where is the sqlite database stored?

### DISCUSSION THREAD

**Discussion by kartikchandran** [vzhd1701/evernote-backup#36] [Q&A] [answered]
I'd like to move the sqlite database to an external drive to save space on my hard drive. Could someone help me with where this file is located? Once I move it, will `evernote-backup sync` automatically find it, or will I need to specify its location? If the latter, how do I do that?

Thanks!
Kartik

**Comment by Drahydel** [comment]
Most of EN's local data is stored in %Appdata%\Roaming\Evernote\resource-cache where You can find a folder UserXXX\. Move that folder to a folder on Your hard drive and make symbolic directory link (by using MKLINK /D) from in ...\resource-cache to the new folder.

**Comment by vzhd1701** [comment, marked as answer]
`evernote-backup` stores database in en_backup.db file in the working directory by default. You can change this by using `--database` option.
